tests/phpunit/tests/functions/registerBlockPattern.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * Test register_block_pattern().
+ *
+ * @group block-patterns
+ * @covers ::register_block_pattern
+ * @covers WP_Block_Patterns_Registry::register
+ */
+class Tests_Functions_RegisterBlockPattern extends WP_UnitTestCase {
```

Comment:
`register_block_pattern()` is a wrapper for `WP_Block_Patterns_Registry::register()`. I'd suggest creating a test class for `WP_Block_Patterns_Registry` instead.

Reply:
PR updated to do just that.